CNV#44000: add candidate channel to update docs (#88599)
openshift/openshift-docs: modules/virt-early-access-releases.adoc

Reviewer: LGTM

Reviewer 2: LGTM

// Module included in the following assemblies:
//
// * virt/updating/upgrading-virt.adoc

:_mod-docs-content-type: CONCEPT
[id="virt-early-access-releases_{context}"]
= Early access releases

> The *candidate-{product-version}* channel offers unsupported early access to releases as soon as they are built. Releases present only in candidate channels

Author: @tal-hason I have changed the text of this module substantially; please take another look and let me know if it looks better to you. Thanks! (Note: we generally don't document dev preview features in the product docs, so I left out mention of the dev-preview channel)

// Module included in the following assemblies:
//
// * virt/updating/upgrading-virt.adoc

:_mod-docs-content-type: CONCEPT
[id="virt-early-access-releases_{context}"]
= Early access releases

> You can gain access to builds in development by subscribing to the *candidate* update channel for your version of {VirtProductName}. These releases have not been fully tested by Red Hat and are not supported, but you can use them on non-production clusters to test capabilities and bug fixes being developed for that version.

Reviewer 3: I've not experienced the issue, but a RH SSG entry exists: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

Red Hat > Red{nbsp}Hat

> The *stable* channel, which matches the underlying {product-title} version and is fully tested, is recommended for production systems. It is possible to switch between the *stable* and *candidate* channels in Operator Hub. However, updating from the *candidate* channel to the *stable* channel is not tested by Red Hat.

Reviewer 3: ```suggestion
The *stable* channel, which matches the underlying {product-title} version and is fully tested, is recommended for production systems. You can switch between the *stable* and *candidate* channels in Operator Hub. However, updating from the *candidate* channel to the *stable* channel is not tested by Red{nbsp}Hat.
```

> The *stable* channel, which matches the underlying {product-title} version and is fully tested, is recommended for production systems. It is possible to switch between the *stable* and *candidate* channels in Operator Hub. However, updating from the *candidate* channel to the *stable* channel is not tested by Red Hat.

Reviewer 3: ```suggestion
The *stable* channel, which matches the underlying {product-title} version and is fully tested, is suitable for production systems....
```

![Screenshot From 2025-04-09 14-52-58](https://github.com/user-attachments/assets/ca477b22-7ed4-408e-89ce-b007df16f18d)
